docs/content/en/samples/renderers/kustomize.yaml

```diff
@@ -0,0 +1,6 @@
+manifests:
```

Comment:
This should be in docs-v2/ dir (not docs/)

Reply:
Resolved. I had to make additional changes in `TestParseSamples` and all docs-v2/samples